workflow/rules/report.smk

```diff
@@ -1,21 +1,19 @@
-rule igv_report:
+rule vcf_report:
     input:
-        bcf="results/merged-calls/{group}.{event}.fdr-controlled.bcf",
         ref="resources/genome.fasta",
-        ref_idx="resources/genome.fasta.fai",
-        bams=lambda w: get_group_bams(w),
-        bais=lambda w: get_group_bams(w, bai=True)
+        bams=lambda w: get_batch_bams(w),
+        bcfs=lambda w: expand("results/merged-calls/{group}.{{event}}.fdr-controlled.bcf", group=get_report_batch(w))
     output:
-        report("results/igv-report/{group}.{event}.html", caption="../report/calls.rst", category="Variant calls")
-    log:
-        "logs/igv-report/{group}.{event}.log"
+        report(directory("results/vcf-report/{batch}.{event}/"), htmlindex="index.html", caption="../report/calls.rst", category="Variant calls")
     params:
-        os.path.join(os.path.dirname(workflow.snakefile), "resources/igv-report-template.html")
+        bcfs=lambda w: expand("{group}=results/merged-calls/{group}.{event}.fdr-controlled.bcf", group=get_report_batch(w), event=w.event),
+        bams=lambda w: get_batch_bams(w, True),
+        format_field = "-f DP AF OBS",
+        info_field = "-i PROB_FFPE_ARTIFACT PROB_PRESENT PROB_ARTIFACT PROB_ABSENT",
```

Comment:
As `event` is a wildcard in this rule, I would use it in specifying which INFO fields to use, here. Users might change the name of the event in the `config.yaml`:
```suggestion
       info_field = lambda w:
           "-i PROB_FFPE_ARTIFACT PROB_{event} PROB_ARTIFACT PROB_ABSENT".format(
               event=w.event.upper()
           ),
```

Reply:
Thanks! That's a good point. Only the expression needs to be a bit diffrent as the varlociraptor events need to be extracted from the config.
It should be something like `lambda wc: "PROB_FFPE_ARTIFACT {} PROB_ARTIFACT PROB_ABSENT".format(' '.join(expand("PROB_{events}", events=config["calling"]["fdr-control"]["events"][wc.event]["varlociraptor"])))`

Reply:
Ah, yes. I was still too focused on my use case. It's even better this way, more general!